src/distance.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from scipy.spatial import distance
+import cv2
+
+class Distance:
+    
+    def __init__(self, query_img, db_img):
+        self.query_img = query_img
+        self.db_img = db_img
+#        self.db_img_filename = db_img_filename
+    
+    def method(self):
+        
+        return self.method
+    
+    def distance(self):
+        
+        return self.distance
+    
+    def  db_img_filename(self):
+        
+        return self.db_img.filename
+    
+
+    
+    def calc_dist(self, similarity_method):
+        """
+        Calc the desired distance
+        
+        """
+        query_hist = self.query_img.histogram
+        
+        
+        
+        db_hist = self.db_img.histogram
+        
+        
+        if similarity_method == "euclidean":
+            self.method = "euclidean"
+            self.distance = distance.euclidean(query_hist, db_hist)
+            return self.distance
```

Comment:
I think this should be a function

`
def calc_euclidean(image1, image2):
    return distance.euclidean(image1, image2)
`

Same for the rest of distance functions